server/service/src/sync/api/core.rs

```diff
@@ -59,25 +42,40 @@ pub enum SyncApiV5CreatingError {
 }
 
 impl SyncApiV5 {
-    pub fn new(
+    pub fn new_settings(
         settings: &SyncSettings,
         service_provider: &ServiceProvider,
         sync_version: u32,
-    ) -> Result<Self, SyncApiV5CreatingError> {
+    ) -> Result<SyncApiSettings, SyncApiV5CreatingError> {
```

Comment:
SyncApiSettings needs to be sent to omSupply central for authentication (for hardware id and credentials, thus it's shape was refactored and SyncApiCredentials was removed